apollo-compiler/src/main/kotlin/com/apollostack/compiler/ClassNames.kt

```diff
@@ -10,4 +12,16 @@ object ClassNames {
   val COLLECTIONS: ClassName = ClassName.get(Collections::class.java)
   val ARRAYS: ClassName = ClassName.get(Arrays::class.java)
   val QUERY: ClassName = ClassName.get(Query::class.java)
+  val OBJECT: ClassName = ClassName.get(Object::class.java)
+  val MAP: ClassName = ClassName.get(Map::class.java)
+  val HASH_MAP: ClassName = ClassName.get(HashMap::class.java)
+
+  fun parameterizedMap(firstTypeArgument: TypeName, secondTypeArgument: TypeName): TypeName =
```

Comment:
nit: what about appending `of` at the end to match the other Kotlin factory methods? eg.: `parameterizedMapOf()`